src/FFI.zig

```diff
@@ -12,168 +12,162 @@ const Reporter = @import("Reporter.zig");
 
 const Self = @This();
 
-const basic_types = std.ComptimeStringMap(
-    o.ObjTypeDef,
-    .{
-        .{ "u8", .{ .def_type = .Integer } },
-        .{ "i8", .{ .def_type = .Integer } },
-        .{ "u16", .{ .def_type = .Integer } },
-        .{ "i16", .{ .def_type = .Integer } },
-        .{ "i32", .{ .def_type = .Integer } },
+const basic_types = std.StaticStringMap(o.ObjTypeDef).initComptime(.{
+    .{ "u8", .{ .def_type = .Integer } },
+    .{ "i8", .{ .def_type = .Integer } },
+    .{ "u16", .{ .def_type = .Integer } },
+    .{ "i16", .{ .def_type = .Integer } },
+    .{ "i32", .{ .def_type = .Integer } },
 
-        // Could it be > 32bits one some systems?
-        .{ "c_int", .{ .def_type = .Integer } },
+    // Could it be > 32bits one some systems?
+    .{ "c_int", .{ .def_type = .Integer } },
 
-        .{ "c_uint", .{ .def_type = .Float } },
-        .{ "u32", .{ .def_type = .Float } },
-        .{ "i64", .{ .def_type = .Float } },
-        .{ "f32", .{ .def_type = .Float } },
-        .{ "f64", .{ .def_type = .Float } },
+    .{ "c_uint", .{ .def_type = .Float } },
+    .{ "u32", .{ .def_type = .Float } },
+    .{ "i64", .{ .def_type = .Float } },
+    .{ "f32", .{ .def_type = .Float } },
+    .{ "f64", .{ .def_type = .Float } },
 
-        .{ "u64", .{ .def_type = .UserData } },
-        .{ "usize", .{ .def_type = .UserData } },
+    .{ "u64", .{ .def_type = .UserData } },
+    .{ "usize", .{ .def_type = .UserData } },
 
-        .{ "bool", .{ .def_type = .Bool } },
+    .{ "bool", .{ .def_type = .Bool } },
 
-        .{ "void", .{ .def_type = .Void } },
-        .{ "anyopaque", .{ .def_type = .Void } },
-    },
-);
+    .{ "void", .{ .def_type = .Void } },
+    .{ "anyopaque", .{ .def_type = .Void } },
+});
```

Comment:
The reformatting is due to the last `,` being removed. Add them back and the diff will disappear.